src/js/errors.js

```diff
@@ -0,0 +1,15 @@
+var Wand = (function(wand) {
```

Comment:
Excellent. I was thinking of proposing this as a separate file - thanks for doing it! I haven't looked into the best way to build error handlers or how other libraries do it, but this seems to make a lot of sense to me.